### DISCUSSION TITLE
"BG Missed Readings" on Pixel 6 Pro

### DISCUSSION THREAD

**Discussion by Grizzly3369** [NightscoutFoundation/xDrip#2069] [Q&A]
Getting "BG Missed Reading" intermittently esp thru the night.
Here are my System Status Pages:
![Screenshot_20220406-142130](https://user-images.githubusercontent.com/57366527/162105258-3d987ea9-b334-409d-8874-d6371f78d6ae.png)
![Screenshot_20220406-142153](https://user-images.githubusercontent.com/57366527/162105308-bece57cb-5f8f-45bb-9271-bbb4425f27d6.png)
Still have BG# on xDrip home screen though and Notifications Bar on P6Pro.

**Comment by Navid200** [comment]
Are you asking why you have connectivity issues?
Or, are you asking why you don't get an alert when you have missed readings?

**Comment by jbr7rr** [comment]
Hi I just updated this https://github.com/NightscoutFoundation/xDrip/pull/2121 with a test build, if you like you can test to see if this fixes your problems (this was previously tested by me also on an pixel 6 pro).

But first and foremost make sure you have followed the steps outlined here:
https://navid200.github.io/xDrip/docs/Intermittent.html

Also my fix needs “Allow OB1 unbonding” unchecked.

Can you report back your findings?
